### DISCUSSION TITLE
linux docker如何多开本项目

### DISCUSSION THREAD

**Discussion by Tosd0** [lss233/chatgpt-mirai-qq-bot#604] [Q&A]
如题，当我在另一个路径下使用release页面的安装脚本时，原来的docker image就会被替换成现在的，之前开的docker就不能继续运行了，无法做到两边同时运行。请问下有没有方法让两个分开运行？感谢！

**Comment by lss233** [comment]
不会啊，找个新的位置执行安装脚本就行了

**Reply by Tosd0** [reply]
我遇到的是，当我已经有一个项目之后，重新运行安装命令，他docker似乎是重新安装（recreate），并不会保留两个。
图一：运行前，已经登陆qq，并可正常处理消息
图二图三：在新目录下运行安装命令（包含docker images命令）
图四：再次打开日志，已经被覆盖。即便去原来的目录下打开日志也会是新的，没有登录的，而且机器人掉线
图五：docker ps命令

![147A47D4-9E64-4186-A9A6-CC4D60FC288F](https://user-images.githubusercontent.com/65720409/231492807-23a878de-570c-4d05-abe7-052295d94fc9.jpeg)

![A16F4C09-CF74-4F69-A542-8627690BBE79](https://user-images.githubusercontent.com/65720409/231492823-3d7c917e-d22a-4f22-85c9-6d85dae51817.jpeg)

![2EA22B5D-FB2D-4786-AAC5-DCDB6A619C34](https://user-images.githubusercontent.com/65720409/231492827-e1b0314b-ccaf-42d8-bc86-3873b23fcdfe.jpeg)

![DCEF63B5-4684-45F7-9880-B3658B0288D6](https://user-images.githubusercontent.com/65720409/231492838-8741a8bf-dfdf-4097-951e-ae236d9ef32f.jpeg)

![B55415D9-A517-49FB-81BA-7225D84F8AC3](https://user-images.githubusercontent.com/65720409/231494634-6ff4f617-9682-436c-9e01-8d4a407a7c53.jpeg)
